Assets/Scripts/CombatZone.cs

```diff
@@ -47,13 +47,16 @@ private void SetCombatPositions()
     {
         for (int i = 0; i < EnemyPositions.Length; i++)
         {
-            Enemies[i].transform.position = EnemyPositions[i].transform.position;
+            GameObject enemy = Enemies[i];
+            enemy.transform.position = EnemyPositions[i].transform.position;
+            enemy.GetComponent<Combatant>().ShowHealth();
```

Comment:
Right so as I mentioned here https://github.com/The-GameDJs/CostumeCurse/pull/31#discussion_r518044369, I think it'd be more elegant to use a proper intialization method. Although, you could just rename `ShowHealth` to something like `EnterCombat` (or something alike) if we don't expect to need to call a `ShowHealth` method in another context.

Reply:
I put it here because I was only thinking showing/removing elements in the CombatZone. You make a great point though! Will fix it!